examples/drupal_loadtest.rs

```diff
@@ -131,13 +131,13 @@ async fn drupal_loadtest_login(client: &mut GooseClient) {
                         Some(f) => f,
                         None => {
                             eprintln!("no form_build_id on page: /user page");
-                            client.set_failure(&GooseMethod::GET, "/user");
+                            client.set_failure(&response.request);
                             return;
                         }
                     };
 
                     // Log the user in.
-                    let uid = rand::thread_rng().gen_range(3, 5_002);
+                    let uid: usize = rand::thread_rng().gen_range(3, 5_002);
```

Comment:
nit - unrelated change (pedantic I know 🙃)

Reply:
Whoops, this slipped in when I was testing various solutions -- in one, this was required.